### DISCUSSION TITLE
v200251

### DISCUSSION THREAD

**Discussion by ipdae** [planetarium/lib9c#3035] [Release]
# Summary
- PoS 준비를 위한 액션이 포함됩니다.
- 스테이킹 일부 사항이 변경사항에 포함되어있습니다.

- Release Date : 2024/11/15

- 시즌 상황 : 챔피언십 기간이라 꼭 필수 배포일지 확인이 필요합니다.

# Changes
## Backward incompatibility changes 
-  Changed Staking action #1385 

## API Changes
- Add PoS action #1385 

# E2E Tests
- [ ] 전체
- [ ] 부분
  - [x] 스테이킹

# Release
-https://github.com/planetarium/libplanet/releases/tag/5.4.0

**Comment by ipdae** [comment]
#1385 는 액션이 하드포킹되는데 그냥 나가도 되나요?

**Comment by riemannulus** [comment]
Staking 관련 체크가 필요해 보입니다.

**Comment by Junyong-Suh** [comment]
현재 챔피언쉽 진행중인데 나가도 될까요?
